Tasks/Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Tasks
+{
+    /// <summary>
+    /// Calculator
+    /// </summary>
+    public class Calculator
+    {
+        /// <summary>
+        /// Calculate operations
+        /// </summary>
+        /// <param name="a">
```

Comment:
param name a - doesn's give me any understanding of meaning. Please change to more meanengful name